### PR TITLE
WIP: Pick up changes in base image

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
See: https://github.com/openshift/sti-base/pull/5

**Comment by jhadvig** [comment]
@mfojtik :+1: 
I've put couple of comments to your sti-base PR. They basically apply also to this changes, since we are moving the logic from this image to the base one. 
Pls check it :)

File: 0.10/Dockerfile.rhel7 [hunk 1]
```diff
@@ -1,38 +1,16 @@
...
-ENV HOME     /opt/nodejs
-ENV PATH     $HOME/bin:$PATH
-
-ENV STI_NODEJS_VERSION 0.10
-
-WORKDIR     /opt/nodejs/src
-
-USER nodejs
+# Copy the nodejs scripts and build tools into /opt/openshift directory
+COPY ./nodejs/ /opt/openshift/
```
Comment (jhadvig) [inline 0.10/Dockerfile.rhel7]: @mfojtik please make this consistent to the centos Dockerfile

**Comment by mfojtik** [comment]
[test]

**Comment by jhadvig** [comment]
LGTM

File: 0.10/Dockerfile.rhel7 [hunk 2]
```diff
@@ -1,38 +1,16 @@
...
-RUN yum install -y --setopt=tsflags=nodocs yum-utils wget which lsof \
-	unzip tar bsdtar procps-ng gettext which gcc-c++ automake autoconf \
-	curl-devel openssl-devel zlib-devel libxslt-devel libxml2-devel gdb \
-	mysql-libs mysql-devel postgresql-devel sqlite-devel && \
-    yum clean all -y
+ENV STI_NODEJS_VERSION 0.10
 
-RUN yum-config-manager --enable rhel-server-rhscl-7-rpms && \
-    yum-config-manager --enable rhel-7-server-optional-rpms && \
+RUN yum install -y \
```
Comment (jhadvig) [inline 0.10/Dockerfile.rhel7]: This should be handled by the yum-config-manager, as it was before

Comment (mfojtik) [inline 0.10/Dockerfile.rhel7]: yeah, I was fast with copy&paste. good catch

**Comment by mfojtik** [comment]
how these tests are passing is a magic for me, the Hub does not have updated sti-base image yet...

**Comment by mfojtik** [comment]
[test]

**Comment by soltysh** [comment]
:+1:

**Pull request merged**